lib/prelude/preloadable.rb

```diff
@@ -22,6 +22,12 @@ def prelude_methods
         @prelude_methods ||= {}
       end
 
+      # Copy parent prelude methods to subclasses
```

Comment:
Another option might be to search through the ancestors when looking up the prelude method:

https://github.com/composerinteralia/prelude/blob/91cc94a8fef5156fc2f528ba7420725768a844bf/lib/prelude/preloader.rb#L9